packages/url-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@aws-sdk/url-parser-browser",
```

Comment:
The values in `homepage` and `repository.directory` also need to be updated.

Reply:
Good point. Addressed in https://github.com/aws/aws-sdk-js-v3/pull/1903/commits/b9e068bd5d5102597920dfb751dedeafe40d8926